example/imageSender/README.md

```diff
@@ -22,4 +30,30 @@
     |--- source
     +--- CMakeLists.txt
 ```
-7. Use CMAKE to trigger the build
\ No newline at end of file
+9. Use CMAKE to trigger the build
+
+![image](https://github.com/ni/adas-replay-hil-simulator-sdk/assets/95634231/0cdd8b24-f1fc-4f5b-948c-98419b2f7739)
+
+10. From the generated ```Build``` folder select the solution file 
```

Comment:
This is a VS specific reference. We should refactor for standard cmake or VS Code.